ui/app/templates/components/mfa-method-list-item.hbs

```diff
@@ -0,0 +1,43 @@
+<LinkedBlock class="list-item-row" @params={{array "vault.cluster.access.mfa.methods.method" @model.id}}>
+  <div class="level is-mobile">
+    <div class="level-left">
+      <div class="is-flex-row">
+        <Icon @size="24" @name={{@model.type}} class="has-text-grey" />
+        <div>
+          <span class="has-text-weight-semibold has-text-black">
+            {{if (eq @model.type "totp") (uppercase @model.type) @model.type}}
+          </span>
+          <span class="tag has-left-margin-xs">
+            {{@model.id}}
+          </span>
+          <div class="has-top-margin-xs">
+            <code class="is-size-9">
+              Namespace:
+              {{@model.namespace_id}}
```

Comment:
I double checked the response from both `/identity/mfa/methods?list=true` and `/identity/mfa/method/:id` and it's being returned as `root` in the dev environment so I think it will always have a value.